packages/data-transformation/fhir-to-csv/main.py

```
@@ -25,80 +23,73 @@
 def transform_and_upload_data(
     input_bucket: str,
     output_bucket: str,
-    api_url: str,
-    job_id: str,
     cx_id: str,
-    patient_id: str | None,
+    patient_id: str,
+    job_id: str,
     input_bundle: str | None,
 ) -> list[tuple[str, str, str]]:
     patient_ids_and_bundle_keys = []
```

Ruff (0.12.2)

Local variable `patient_ids_and_bundle_keys` is assigned to but never used

Remove assignment to unused variable `patient_ids_and_bundle_keys`

(F841)